charts/opentelemetry-collector/values.yaml

```diff
@@ -25,6 +25,7 @@ presets:
   logsCollection:
     enabled: false
     includeCollectorLogs: false
+    storeOffsets: false
```

Comment:
Can you add a comment describing what this setting does and the security implications of using this setting?